db/seeds/app_metadata.yml

```diff
@@ -8,6 +8,9 @@
 ci:
   "/insights/platform/cost-management":
     gcp_service_account: test-billing-service-account@cloud-billing-292519.iam.gserviceaccount.com
+    aws_wizard_account_number: "8675309"
```

Comment:
I'm guessing this is fine with Cost to make static?  Should it be an env variable?

Reply:
Well, I will substitute it out eventually for the real account, but that is a dummy number for now (jenny jenny...)

The number is public and in the docs + wizard, so it will probably be ok to be public.

Reply:
I feel sorry for the people who actually had that phone number.

Does this need to be changed before it's merged, or does it not matter?

Reply:
Nope, it will be fine for now. I will update it when the superkey worker is deployed and ready, that way we aren't accidentally creating resources in the meantime.